Makefile

```diff
@@ -140,44 +140,51 @@ envtest:
 test: generate manifests envtest ## Run tests.
 	KUBEBUILDER_ASSETS=$(KUBEBUILDER_ASSETS) go test ./... -coverprofile cover.out $(GO_TEST_GINKGO_ARGS)
```

Comment:
If we have 2 targets for unit and integration tests, why we need to implement the test target? it could be an empty target depending on the other targets:

```
test: unit-tests integration-tests

unit-tests:
    ...

integration-tests:
    ...
```